pkg/vere/io/http.c

```diff
@@ -2915,6 +3108,14 @@ u3_http_io_init(u3_pier* pir_u)
     u3z(now);
   }
 
+  //Setup spin stack
+  htd_u->stk_u = u3t_sstack_open();
+  
+  if ( NULL == htd_u->stk_u ) {
+    u3l_log("failed to open spin stack");
+    u3_pier_bail(car_u->pir_u);
+  }
```

Comment:
let's move this to `u3_http_io_talk()` to avoid race conditions, and make it a soft error (just print).

Reply:
done